shared/src/sdk/eth_bridge/validator_set.rs

```diff
@@ -285,16 +288,22 @@ pub async fn query_validator_set_update_proof<C, IO: Io>(
         .eth_bridge()
         .read_valset_upd_proof(client, &epoch)
         .await
-        .unwrap();
+        .map_err(|err| {
+            let msg =
```

Comment:
I'm noticing this pattern occur a lot. While not necessary, maybe we add another macro that both prints and returns the printed string. The hardest part will be coming up with a good name for it...